cmd/modern/main_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package main
+
+import (
+	"errors"
+	"github.com/microsoft/go-sqlcmd/internal/cmdparser"
+	"github.com/microsoft/go-sqlcmd/internal/cmdparser/dependency"
+	"github.com/microsoft/go-sqlcmd/internal/output"
+	"github.com/microsoft/go-sqlcmd/internal/pal"
+	"github.com/microsoft/go-sqlcmd/internal/test"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestMainStart(t *testing.T) {
+	os.Args[1] = "--help"
+	main()
```

Comment:
>main() [](http://example.com/codeflow?start=1&length=6)

"doesn't panic" seems like a very minimal test. Couldn't this test verify that rootCmd is constructe with proper values?